index.md

```diff
@@ -20,21 +20,48 @@ permalink: /
 
 </div>
 
-## New to JSON Schema?
+## What now?
 
-Learning a new specification can be daunting.
+Learn, Get help, Shape the Community, Chat, with the JSON Schema team and Community!
 
-You should read our [getting started guide](/learn/getting-started-step-by-step)!
+<div class="wrapper text-center buttons">
+    <a class="button border button-center" href="/learn/getting-started-step-by-step">👋 Getting Started</a>
+    <a class="button border button-center" href="/slack" target="_blank"><img class="small-svg-logo" src="/assets/logo-slack.svg" height="1.3em" width="1.3em"> Open JSON Schema Slack</a>
+    <a class="button border button-center" href="https://github.com/json-schema-org/community/discussions" target="_blank">💬 Community Discussions</a>
+</div>
+
+## Regular Activities
+
+We hold weekly Office Hours and twice monthly Open Community Working Meetings.
+
+<div class="wrapper text-center buttons">
+    <a class="button border button-center" href="https://github.com/json-schema-org/community/discussions/34" target="_blank">🧑‍💻 Office Hours</a>
+    <a class="button border button-center" href="https://github.com/json-schema-org/community/discussions/35" target="_blank">👷 Open Community Working Meetings</a>
+</div>
+
+Office Hours are every Tuesday at 15:00 UTC.
+
+Open Community Working Meetings follow two patterns:
+- First Friday of the month at 20:00 UTC.
+- Third Friday of the month at 15:00 UTC.
+
+## Need more?
+
+We have our other [learning resources](/learn), including the [Understanding JSON Schema documentation](/understanding-json-schema).
 
-You can also see our other [learning resources](/learn).
+## About Our Community
 
-### Got questions?
+We have an active and growing community. All are welcome to be part of our community, help shape it, or simply observe.
+
+We want keep our community welcoming and inclusive, so please read our [JSON Schema Organizational Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md). (This is a combination of the Contributor Covenant and IETF BCP 54.)
```

Comment:
```suggestion
We want to keep our community welcoming and inclusive, so please read our [JSON Schema Organizational Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md). (This is a combination of the Contributor Covenant and IETF BCP 54.)
```

Reply:
Github's email produces an alert here:

> This file contains bidirectional Unicode text that may be interpreted or compiled differently than what appears below. To review, open the file in an editor that reveals hidden Unicode characters. [Learn more about bidirectional Unicode characters](https://github.co/hiddenchars)

Reply:
> Github's email produces an alert here:
> 
> > This file contains bidirectional Unicode text that may be interpreted or compiled differently than what appears below. To review, open the file in an editor that reveals hidden Unicode characters. [Learn more about bidirectional Unicode characters](https://github.co/hiddenchars)

Which email?
I don't believe there are any bidirectional unicode text. I have added some unicode characters.
I don't see the same warning.
You should also see the warning when viewing the file on GitHub, and I do not.

Reply:
The github notification emails (and it also shows up in the slack notifications).

I suspect it's a bug with how github is generating diffs, and it's triggering their own heuristic.